addon/components/x-toggle/component.js

```diff
@@ -37,57 +70,29 @@ export default Ember.Component.extend({
     return this.get('elementId') + '-x-toggle';
   }),
 
-  wasToggled: on('init', observer('toggled', function () {
-    var toggled = this.get('toggled');
-    var offIndex = this.get('offLabel').indexOf(':');
-    var onIndex = this.get('onLabel').indexOf(':');
-    var offState = offIndex > -1 ? this.get('offLabel').substr(offIndex + 1) : false;
-    var onState = onIndex > -1 ? this.get('onLabel').substr(onIndex + 1) : true;
-
-    this.sendAction('toggle', toggled, this.get('name'));
-
-    if (toggled === false) {
-      this.set('value', offState);
-    } else {
-      this.set('value', onState);
+  actions: {
+    onClick(e) {
+      e.stopPropagation();
```

Comment:
i think I'm probably missing something. But should we trigger an event with the current checked value when there is a click?